samples/unit-tests/interaction/panning/demo.js

```diff
@@ -0,0 +1,64 @@
+QUnit.test('#14426: Vertical panning after zooming', assert => {
+    const chart = Highcharts.chart('container', {
+        chart: {
+            type: 'column',
+            zoomType: 'xy',
+            panKey: 'shift',
+            panning: {
+                enabled: true,
+                type: 'xy'
+            }
+        },
+        xAxis: {
+            type: 'category'
+        },
+        series: [{
+            data: [{
+                name: "A",
+                y: 1.2
+            }, {
+                name: "B",
+                y: 4.02
+            }, {
+                name: "C",
+                y: 1.92
+            }]
+        }]
+    });
+
+    const offset = Highcharts.offset(chart.container);
+
+    // Zoom
+    chart.pointer.onContainerMouseDown({
+        type: 'mousedown',
+        pageX: 250 + offset.left,
+        pageY: 50 + offset.top,
+        target: chart.container
+    });
```

Comment:
FYI: Instead of playing with `offset` you can use `TestController` :)

Reply:
I got "inspired" by the other panning tests :)